pkg/resource/cluster/hooks.go

```diff
@@ -265,12 +265,12 @@ func (rm *resourceManager) newMemoryDBClusterUploadPayload(
 	return res
 }
 
-// clusterActive returns true when the status of the given Cluster is set to `active`
-func (rm *resourceManager) clusterActive(
+// isClusterAvailable returns true when the status of the given Cluster is set to `active`
```

Comment:
you haven't updated this code comment with the new value

Reply:
Fixed